client/src/app.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.stat={
```

Comment:
I believe this is a typo?
```suggestion
     this.state={
```